src/store/registration/saga.ts

```diff
@@ -169,6 +172,30 @@ export function* updateProfile(action) {
       yield put(setFirstTimeLogin(true));
       yield call(completeUserLogin);
       yield put(setStage(RegistrationStage.Done));
+
+      const inviterMatrixId = response.response.inviter.matrixId;
+      const inviterUserId = response.response.inviter.id;
+      const userData = yield call(getZEROUsersAPI, [inviterMatrixId]);
+
+      const normalizeUserData = (userData) => {
```

Comment:
Does this need to be a function? Can we just create the  user struct right here? We only use it in one place.

Side note: do we truly need this here? Doesn't creating the conversation already go through paths that would fetch the rest of the user data anyway?

Reply:
Updated to use a struct rather than function. I believe we do need this here, if removed, there is no user data.

Reply:
What happens when a we normally create a new conversation with a user we've never loaded previously? Do we not have code that ends up fetching the user data we need to render the user?

Reply:
The user data is available from the store when searching for the user you are about to start a conversation with. Until you search, the user data is empty. That was why I added this here, but maybe there's a better solution?

Reply:
Sure, but if a different user creates a conversation with you and you receive the real time event we have to fetch the info. I just assumed the conversation handling went through the same mapping. Perhaps it doesn't.